server/lib/scheduler/scheduler.cancelJob.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @description Cancel job
+ * @param {string} jobName - Job name to cancel.
+ * @example
+ * scheduler.cancelJob('my-job');
+ */
+function cancelJob(jobName) {
+  this.nodeSchedule.cancelJob(jobName);
```

Comment:
I can't see this method in the doc from node-schedule? https://github.com/node-schedule/node-schedule